core/domain/feature_flag_services.py

```diff
@@ -264,7 +266,7 @@ def evaluate_all_feature_flag_configs(
     feature_flags = get_all_feature_flags()
     for feature_flag in feature_flags:
         feature_flag_status = is_feature_flag_enabled(
-            feature_flag.name, feature_flag, user_id)
+            feature_flag.name, user_id=user_id, feature_flag=feature_flag)
```

Comment:
I don't think you need the `user_id=` part here, ditto in a bunch of other places

Reply:
Done.